runtime/src/in_mem_accounts_index.rs

```diff
@@ -264,6 +271,42 @@ impl<T: IndexValue> InMemAccountsIndex<T> {
         result
     }
 
+    pub(crate) fn flush(&self) {
+        let flushing = self.flushing_active.swap(true, Ordering::Relaxed);
+        if flushing {
+            // already flushing in another thread
+            return;
+        }
+
+        self.flush_internal();
+
+        self.flushing_active.store(false, Ordering::Relaxed);
```

Comment:
Since `flushing_active` is like a thread-safe flag, and this part lowers the flag, I think this Ordering should be `Ordering::Release`. This ensures no loads/stores get reordered _after_ this store. I.e. this 'done flushing' only happens when the flushing is done, and nothing happens afterwards.